### PR TITLE
Arial font only + UG update

### PR DESCRIPTION
**Pull request by NNpanpan**
Can change to any fonts that would work on any system...

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -686,12 +686,13 @@ Receive reminders for deadlines and events the next day.
...
 |  `*task* delete MOD_CODE ID_NUMBER`
+|  `*task* done /code MOD_CODE /id ID_NUMBER`
 |  `*task* edit MOD_CODE ID_NUMBER [/desc DESCRIPTION] [/on DATE] [/at TIME]` +
    `*task* edit MOD_CODE ID_NUMBER [/desc DESCRIPTION] /on non`
-|  `*task* find [/date DATE] [/month MONTH] [/year YEAR]` +
-   `*task* find /desc TASK_NAME` +
+|  `*task* find /desc TASK_NAME` +
    `*task* find /upcoming`
 |  `*task* list`
+|  `*task* search [/date DATE] [/month MONTH] [/year YEAR]`
```
Comment (alushingg) [inline docs/UserGuide.adoc]: Day i think instead of date

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -686,12 +686,13 @@ Receive reminders for deadlines and events the next day.
 |  `*mod* view INDEX` +
    `*mod* view MOD_CODE`
 
-.5+|Task
+.6+|Task
```
Comment (alushingg) [inline docs/UserGuide.adoc:689]: Can help change the task add the /task to /desc

File: docs/UserGuide.adoc (hunk 1 above)
Comment (NNpanpan) [reply docs/UserGuide.adoc]: fixed

File: docs/UserGuide.adoc (hunk 2 above)
Comment (NNpanpan) [reply docs/UserGuide.adoc:689]: yah got it

**Pull request merged**